src/main/java/Duke.java

```diff
@@ -1,10 +1,149 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
+        String input;
+        int byeIndicator = 1;
```

Comment:
Overall, the code is very readable. However, the main method is quite long. It could be better to apply SLAP coding principles and make multiple methods to handle the many variables in main(). 

In addition, some of the method names are not very clear. For example, what is a traditional task? Perhaps a few lines of comments to explain such names, or finding more apt names could be better.